src/vma/sock/sockinfo.h

```diff
@@ -303,7 +303,10 @@ class sockinfo : public socket_fd_api, public pkt_rcvr_sink, public pkt_sndr_sou
 	bool 			detach_receiver(flow_tuple_with_local_if &flow_key);
 	net_device_resources_t* create_nd_resources(const ip_address ip_local);
 	bool                    destroy_nd_resources(const ip_address ip_local);
-	void			do_rings_migration();
+	void			do_rings_migration(resource_allocation_key &old_key);
+	int			set_ring_attr(vma_ring_alloc_logic_attr *attr);
+	int			set_ring_attr_helper(ring_alloc_logic_attr *sock_attr,
+				  		     vma_ring_alloc_logic_attr *attr);
```

Comment:
please use single line for set_ring_attr_helper()

Reply:
Fixed.